.eslintrc.js

```diff
@@ -35,6 +35,7 @@ module.exports = {
     'import/prefer-default-export': 'off',
     'react-hooks/rules-of-hooks': 'error',
     'react-hooks/exhaustive-deps': 'warn',
+    'react/jsx-props-no-spreading': 'off',
```

Comment:
@musllim  Why are you disabling this rule?

Reply:
prop spreading does not work in typescript with eslint for some reason, so whenever you want to use them you have to ignore that linter rule.

Reply:
Could you elaboraborate which props you are spreading, and why?